test/system/roles/public_health/dashboard/import_verifier.rb

```diff
@@ -301,6 +303,10 @@ def normalize_bool_field(value)
     %w[true false].include?(value.to_s.downcase) ? (value.to_s.downcase == 'true') : nil
   end
 
+  def unformat_enum_field(value)
```

Comment:
I see you've got this defined in two different places (here and also in import_controller), and also something similar as `self.normalize_enums(enums_dict)` in `ValidationHelper`. Is there a way to consolidate? Also, please put a comment describing what this does, why it is needed, and perhaps even where it is used (took me a second to figure that out, with a bit of grepping).

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-87d122d0e0fdbdc566749c16f690698e673bd67021cee51103809dd0f9148af0R13